frontend/public/components/operator-lifecycle-manager/create-operand.tsx

```diff
@@ -0,0 +1,264 @@
+        <div className="form-group">
+          <label className="form-label co-required" htmlFor={field.path}>{field.displayName}</label>
+          { inputFor(field) }
+          { field.description && <span className="help-block text-muted">{field.description}</span> }
+        </div>
+      </div>)}
+      <div className="row">
+        {error && <p className="alert alert-danger co-alert"><span className="pficon pficon-error-circle-o"></span>{error}</p>}
+      </div>
+      <div className="row">
+        <button className="btn btn-primary" type="submit" onClick={submit}>Create</button>
+        <Link className="btn btn-default" to={`/k8s/ns/${props.namespace}/clusterserviceversions/${props.clusterServiceVersion.metadata.name}/${referenceForModel(props.operandModel)}`}>Cancel</Link>
+      </div>
+    </form>
+    <div className="col-md-8">
+      { props.clusterServiceVersion && props.providedAPI && <div>
+        <ClusterServiceVersionLogo
+          displayName={props.providedAPI.displayName}
+          icon={_.get(props.clusterServiceVersion, 'spec.icon[0]')}
+          provider={_.get(props.clusterServiceVersion, 'spec.provider')} />
+        { props.providedAPI.description }
+      </div> }
+    </div>
+  </div>;
+};
+
+/**
+ * Component which wraps the YAML editor to ensure the templates are added from the `ClusterServiceVersion` annotations.
+ */
+export const CreateOperandYAML: React.FC<CreateOperandYAMLProps> = (props) => {
+  const template = _.attempt(() => safeDump(props.sample));
+  if (_.isError(template)) {
+    // eslint-disable-next-line no-console
+    console.error('Error parsing example JSON from annotation. Falling back to default.');
+  }
+
+  return <CreateYAML template={!_.isError(template) ? template : null} match={props.match} showHeader={false} />;
+};
+
+export const CreateOperand: React.FC<CreateOperandProps> = (props) => {
+  const providedAPI = () => providedAPIsFor(props.clusterServiceVersion.data).find(crd => referenceForProvidedAPI(crd) === referenceForModel(props.operandModel));
+  const sample = () => JSON.parse(_.get(props.clusterServiceVersion.data.metadata.annotations, annotationKey, '[]'))
+    .find((s: K8sResourceKind) => referenceFor(s) === referenceForModel(props.operandModel));
+  const [method, setMethod] = React.useState<'yaml' | 'form'>('yaml');
+
+  return <React.Fragment>
+    { props.loaded && <div className="co-create-operand__header">
+      <div className="co-create-operand__header__buttons">
+        <BreadCrumbs breadcrumbs={[
+          {name: props.clusterServiceVersion.data.spec.displayName, path: window.location.pathname.replace('/~new', '')},
+          {name: `Create ${props.operandModel.label}`, path: window.location.pathname},
+        ]} />
+        <div style={{marginLeft: 'auto'}}>{
+          method === 'form' && <button className="btn btn-link" onClick={() => setMethod('yaml')}>YAML Editor</button> ||
```

Comment:
We might eventually make these tabs.

```suggestion
          method === 'form' && <button className="btn btn-link" onClick={() => setMethod('yaml')}>Edit YAML</button> ||
```

Reply:
I tried tabs at first, but it was pretty hard to wrangle the existing `HorizontalNav` components to perform the right way. Also the existing pattern is that tabs show different info, not the same info in different ways, which I thought was confusing to use.